### DISCUSSION TITLE
Architecture requirement

### DISCUSSION THREAD

**Discussion by santoshreddyspy** [StackStorm/st2#5628] [Q&A] [answered]
Hi,

I was looking for a specific deployment to meet a requirement wherein; client env is completely secured.
I've attached the detailed document on the requirement. Can you please check if it is feasible. If yes, can you share the necessary document to achieve.

Regards,
Santosh
[StackStorm solution proposal.docx](https://github.com/StackStorm/st2/files/8537769/StackStorm.solution.proposal.docx)

**Comment by nzlosh** [comment, marked as answer]
I'd recommend contacting a [StackStorm parnter](https://stackstorm.com/partners/) to assist you with any architectural design needs.

**Reply by santoshreddyspy** [reply]
Thanks @nzlosh . 
Can you share few contacts. Above hyperlink doesn't give contact details or procedure to become partner.

**Comment by nzlosh** [comment]
There is a button to click for more information about each partner.  I'd expect that would show how to to get in contact with the partner that interests you.

**Comment by mickmcgrath13** [comment]
Hello Santosh,

I’ve followed up via email.  Looking forward to hearing from you
